types/types.go

```diff
@@ -80,6 +80,8 @@ const (
 
 	LonghornLabelKeyPrefix = "longhorn.io"
 
+	LonghornLabelVolumeAccessMode = "volume-access-mode"
```

Comment:
Suggest moving this together with other labels, ref https://github.com/longhorn/longhorn-manager/blob/master/types/types.go#L85-L104.

Reply:
Thank you. I did not notice that and I will follow the suggestion to move the `LonghornLabelVolumeAccessMode` together with other longhorn labels.